### PR TITLE
Check flag values before printing

### PR DESCRIPTION
**Pull request by tashian**

### CODE REVIEW

**Comment by tashian** [comment]
@maraino I've tested this out with various flag combos, seems to work well.

File: ui/ui.go [hunk 1]
```diff
@@ -85,6 +85,13 @@ func PrintSelected(name, value string, opts ...Option) error {
 	return nil
 }
 
+// Print if the value (eg. a specific flag value) is empty
```
Comment (maraino) [inline ui/ui.go:88]: Comments should have the format `// <method> ...`, in this case `// PrintlnIfEmpty ...`

File: ui/ui.go [hunk 2]
```diff
@@ -85,6 +85,13 @@ func PrintSelected(name, value string, opts ...Option) error {
...
+func PrintlnIfEmpty(s, value string) {
+	if value == "" {
+		Println(s)
+	}
+}
```
Comment (maraino) [inline ui/ui.go:93]: I'm not sure if we should add this method here, it doesn't fit the way other methods work, ideally `ui.Println` should be able to detect `ui.Options` and act accordingly, but that's a bit more difficult. 

I would move this method to `command/ca/init.go` and make it private.

**Review by maraino — CHANGES_REQUESTED** [review]
Looks good, but I would move that method to init.go. 

I can also see if I can made ui.Println to support options so we can do things like 
```go
ui.Println("What's your name?", ui.WithValue(ctx.String("name"))
```

**Comment by tashian** [comment]
OK, I've addressed your feedback. The version with the options would be nice to have. If you want to do that, please add it. Or just approve and I'll merge :D

**Pull request merged**